training/general/src/environments/sklearn-1.1/context/intel-extension/conda_dependencies.yaml

```diff
@@ -0,0 +1,22 @@
+name: sklearn-1.1
+channels:
+- conda-forge
+- anaconda
+dependencies:
+- python=3.9
+- pip=21.3.1
+- pandas~=1.3.0
+- scipy~=1.7.0
+- numpy~=1.21.0
+- scikit-learn-intelex~={{latest-sklearnex-version}}
```

Comment:
@savitamittal1 what do you think about adding this package to sklearn-1.1? If you're in favor then I'll have some other comments on the PR. We could also consider creating something like an azureml-contrib registry at some point, which could include this.